pkg/github/config.go

```diff
@@ -43,28 +46,55 @@ type Config struct {
 	GitHubRunAttempt        int64
 	GitHubJob               string
 	GitHubPullRequestNumber int
+	GitHubPullRequestBody   string
 	GitHubSHA               string
 }
 
+type configDefaults struct {
+	Owner             string
+	Repo              string
+	PullRequestNumber int
+	PullRequestBody   string
+}
+
 func (c *Config) RegisterFlags(set *cli.FlagSet) {
+	d := &configDefaults{}
+
 	githubContext, _ := githubactions.New().Context()
-	owner, repo := githubContext.Repo()
-	prNumber, _ := githubContext.Event["number"].(int)
+	d.Owner, d.Repo = githubContext.Repo()
+
+	data, _ := json.Marshal(githubContext.Event) //nolint:errchkjson //Shouldnt affect defaults
```

Comment:
```suggestion
	data, _ := json.Marshal(githubContext.Event) //nolint:errchkjson //Shouldn't affect defaults
```
Curious what's the purpose of this line. Maybe add a concise note about it.

Reply:
fixed